uptime-kuma/templates/tests/test-connection.yaml

```diff
@@ -6,10 +6,14 @@ metadata:
     {{- include "uptime-kuma.labels" . | nindent 4 }}
   annotations:
     "helm.sh/hook": test
+    {{- with .Values.testPod.annotations }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
 spec:
   containers:
     - name: wget
-      image: {{ .Values.image.registry | default "docker.io" }}/busybox
-      command: ['wget']
+      image: {{ .Values.testPod.image }}
+      command: ["/bin/sh"]
       args: ['{{ include "uptime-kuma.fullname" . }}:{{ .Values.service.port }}']
+      args: ["-c", "cd tmp/ ; wget '{{ include "uptime-kuma.fullname" . }}:{{ .Values.service.port }}'"]
```

Comment:
Sorry, Yeah, I changed this because in some clusters, `wget` can't write to `/`, which causes the test to fail with 
```
Connecting to uptime-kuma:3001 (172.xx.xx.xxx:3001)
wget: can't open 'index.html': Permission denied
```
And if someone choses to use another image that already has wget, you're sure that wget will be able to write to the `tmp` folder and pass without issues...

Reply:
shows that I never used the test container :sweat_smile: (actually tempted to replace the `wget` with `curl` or simply direct the `wget` output to `stdout`, but that can be done whenever)